### ISSUE TITLE
React: No label for - Search edit combo list box

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4195]
## Detailed description
On JAWS/FF, iPhone/Voiceover
 
Issue: Search edit combo list box - No label for this control. 

Expected: give label as "search"

**Comment by andysherman2121** [comment]
@Praveen-sr According to https://www.w3.org/TR/wai-aria-practices-1.1/examples/combobox/aria1.1pattern/listbox-combo.html

An element with a role of combobox doesn't accept an aria label attribute. Can you clarify if this is what you were asking for?

**Comment by Praveen-sr** [comment]
@andysherman2121 As per the aria 1.1 spec, we can use aria-label on the combobox role instead of aria-labelledby where there is no visible label available. It appears that the example has a bug. An issue is raised on the same. Please refer the following link on Github. You can try with "aria-label". 
I will retest it with jaws FF and VO iPhone with Safari and update you.
https://github.com/w3c/aria-practices/issues/886
